Multicluster: kind support (#142)
IBM/the-mesh-for-data: website/content/contribute/code/flow/index.md

make e2e
```

# Building in a multi cluster environment

> As {{< name >}} is can run in a multi-cluster environment there is also a test environment

Reviewer: Why "is can"?

> As the-mesh-for-data is can run in a multi-cluster environment there is also a test environment

Reviewer: ```suggestion
As {{< name >}} can run in a multi-cluster environment there is also a test environment
```
